src/services/services.ts

```diff
@@ -1834,7 +1834,7 @@ namespace ts {
             const fileContents = sourceFile.text;
             const result: TodoComment[] = [];
 
-            if (descriptors.length > 0) {
+            if (descriptors.length > 0 && !isNodeModulesFile(fileName)) {

```

Comment:
The program.isSourceFileFromExternalLibrary cannot be used as it only identifies a file as external library if it is found during the dependency resolution. Since the getTodoComments method checks the file individually it fails when an editor submits a node_modules file as a root file.